### ISSUE TITLE
Bot having issues validating user's group membership

### ISSUE THREAD

**Issue by robertlong** [w3c/ash-nazg#242]
I'm having difficulties with the ipr bot which is failing to validate the group membership of one of my community group's members @mikeskydev

https://labs.w3.org/repo-manager/pr/id/omigroup/gltf-extensions/56

He says that he has joined the community group, which I can verify in the group's member list. He also says has linked his GitHub account to his w3c profile. I've revalidated the PR multiple times and it still fails. I can merge the PR since I can manually prove he's signed the CLA, but I'd like for the bot to work for him in the future. Does anyone have any clues as to what is going on?

**Comment by deniak** [comment]
I found that @mikeskydev actually has two W3C accounts (icosagallery and mikesky).

Initially, the first account was connected to GitHub and he then changed that to the second account (the one participating in the CG). However, the repository manager only keeps track of the first connected account which created the issue.
I've fixed it the record and the PR now validates correctly.

@mikeskydev, are you ok if I delete your first account which doesn't seem to be used?
